governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
+## Op1st Community Governance 1.0 (MVG)
+
+A healthy open source project is one that demonstrates open practices, uses open infrastructure, and cultivates an open culture with the goal of becoming more sustainable.
+
+### Principles
+As a new cloud-native community growing up at the footsteps of massively scaled and successful projects such as Kubernetes and OpenInfra, we want to be careful to learn from those who came before us, while forging our own path into the challenges we've chosen to undertake.
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
+  - This is crucial to fulfilling our mission around open sourcing operations
+2. **Welcoming and respectful** : We intend to be an open, welcoming, and inclusive community.
+Diversity is our strength.
+We have and follow a Code of Conduct.
+  - We currently follow the OpenInfra Foundation Code of Conduct as a member of the OpenInfra Labs.
```

Comment:
Isn't this redundant with the later "Code of Conduct" section? https://github.com/operate-first/community/pull/107/files#diff-fd3edf48a550fc1e06e551c9de0928dc1df51b9aef61139df0bed27ed2d9b468R38

Reply:
> Isn't this redundant with the later "Code of Conduct" section? https://github.com/operate-first/community/pull/107/files#diff-fd3edf48a550fc1e06e551c9de0928dc1df51b9aef61139df0bed27ed2d9b468R38

No, I don't see it as redundant or repetitive. This reference to the Code of Conduct is in a section stating the overarching community principles, one of which is to have and follow a code of conduct.

The other link is the actual section where the governance talks about the specific project Code of Conduct, and acts as the in-document referenced location for any other reference in the document.

(I believe it would even be proper and reasonable to include the entire text to a Code of Conduct within the Governance itself, as part of having an entirely self-referential governance. But I'll save the rest of that discussion for when we get to actually deciding as a community what our Code of Conduct should say and do.)